clients/flash/air-client/src/Main.mxml

```diff
@@ -4,6 +4,7 @@
 			   applicationComplete="applicationCompleteHandler(event)"
 			   preinitialize="preinitializeHandler(event)"
 			   styleName="mainshell"
+				 backgroundAlpha="0"
```

Comment:
This should be attached to the style instead of being hardcoded to the class.

Reply:
Changed to setting styleName="no-background".